source/includes/fact-oplog-background.rst

```diff
@@ -1,7 +1,14 @@
-The :term:`oplog` in the source cluster must be large enough to track
-events that happen during the time it takes to complete the initial
-sync to the destination cluster.
+
+``mongosync`` applies operations in the ``oplog`` to the data on the
+destination cluster.  When older operations roll off the ``oplog``
+on the source cluster, the sync fails and ``mongosync`` exits.
```

Comment:
```suggestion
``mongosync`` applies operations in the source cluster's ``oplog`` to the data on the
destination cluster.  When older operations roll off the ``oplog``
on the source cluster, the sync fails and ``mongosync`` exits.
```
Just a clarification on the first mention of the oplog, since that's how it's referenced in the second sentence (or we can move this reference in the second sentence to the first sentence, whichever is clearer to you).

Also, I think the second sentence is a bit inaccurate. Older operations are free to roll off the oplog as long as mongosync has applied them which, with the 1.4.0 release, it will now do much earlier in the migration (i.e. it does so concurrently as it copies documents in initial sync). I think rephrasing this to something like this would be more accurate:

> When older operations **which mongosync has not applied** roll off the oplog on the source cluster, the sync fails and mongosync exits.

Reply:
Fixed